### ISSUE TITLE
Labels

### ISSUE THREAD

**Issue by afrey15** [famegames/fire-and-ice#75]
I feel like we did not try my label system. It's been like 7 hours, not 7 days.

**Comment by Regulator13** [comment]
I changed your labels back before we agreed to anything, I said you'd have to change them back to what they were. Go ahead, you've got week

**Comment by afrey15** [comment]
I feel like the real irony here is that you've changed half of them to a shade of red.

**Comment by Regulator13** [comment]
Just back to what they were before. I noticed you deleted the interface-change label. It stood for an issue related to the screens outside of the actual game, could we add this again?

**Comment by afrey15** [comment]
I get what it stands for. I don't get why that label would do anything for us

**Comment by Regulator13** [comment]
It helps people easily distinguish which things they can fix (in-game) and which things they'll need to understand an impossibly complicated menu system for

**Comment by afrey15** [comment]
Okay, interface is not the correct word to use for that. I don't think that does anything for us. There are menu changes that don't require extensive knowledge,

**Comment by afrey15** [comment]
The user interacts with the entire game so interface doesn't narrow that down at all. You could change the text, color, and position of buttons probably without understanding the menu system. I bet there are other things too.

**Comment by Regulator13** [comment]
How about a menus label or something then. I kinda doubt you could change the text on a button without understanding the menus. I don't know how to do it and I just read through all the menu objects

**Comment by Regulator13** [comment]
Another request, since we're having discussions about meta stuff on here it'd be nice to have a label to separate game changes from organization/github changes. Not sure what to call it though

**Comment by afrey15** [comment]
If we aren't going to have a system for our labels, I think we should have some kind of criteria before a new one is created. I think a good criteria would be having ten or more previous issues where the label could be applied. That would help show the necessity of the label. I think this discussion about the "interface-change" label hearkens back to my earlier arguments well. I didn't understand what it meant exactly. I probably would have misused it. If people don't use the labels in the same way, they aren't really worth anything. Each label that isn't incredibly clear runs the risk of having people misusing them and needing to explain the proper use, arguably, more hassle than the labels are worth.

**Comment by afrey15** [comment]
I'm don't understand how it helps. Is it that we want to prevent people from wasting time reading issues that they won't be able to handle? We could have all sorts of content-related description labels like `menus`.

**Comment by Regulator13** [comment]
I mean, basically yes exactly that. You're right we don't want to go overboard on labels, I just thought this one would do a good job of splitting the issue types in two. If you really think its not worth it we won't add it. It was just an idea.

**Comment by afrey15** [comment]
I don't think its our job to decide what people can't tackle. Side question: where did you get the idea for the pull-request-submitted label? This one is an out-of-your-way sort of extra step, most issues end up with it on there, and it is done so closely in conjunction with the closing of an issue that it is relatively hardly seen.

**Comment by afrey15** [comment]
For what it's worth. "I think" are part of the first three words of my last comment.

**Comment by Regulator13** [comment]
In regard to the first part yet, but you state the second part like fact.

**Comment by afrey15** [comment]
I guess I'm confused. The second part is not an opinion, unless I'm missing something?

**Comment by Regulator13** [comment]
> This one is an out-of-your-way sort of extra step, most issues end up with it on there, and it is done so closely in conjunction with the closing of an issue that it is relatively hardly seen.

I don't agree. I don't think its an out out-of-your-way extra step. I think it's a necessary step. They are seen for sure, just only if you check the issues frequently. This is your opinion, if it was fact I couldn't disagree

**Comment by afrey15** [comment]
It's out-of-your-way because pulling up the issues is not a step in submitting a pull request. This is an extra step outside of the process. I'm not saying it isn't necessary. In regards to being seen, this label is "relatively hardly seen" because it happens after the issue is nearly complete and closed.

**Comment by Regulator13** [comment]
Alright I get what you're saying, and I think you took my point to state your opinions as opinions so we good.

**Comment by afrey15** [comment]
Yah no I agree. Saying, "It is unnecessary" doesn't really do anything for anyone. I am trying to base my arguments on facts, which is why they don't have the qualifiers. I have probably done this wrong in the past though so I will pay more attention to it.

**Comment by afrey15** [comment]
I have a feeling there won't be much more "meta" things to say about the Github after this discussion ends. But, in regards to your question on a label for it, I was thinking about the team discussion groups that GitHub has. It could be a possibility to move such things there. I would also like to rework the templates a little bit, adding the context header on issues and the expected behavior header on bugs.

**Comment by afrey15** [comment]
I will address the other issues here, but I've got some more thoughts on this one. So one of the changes I made was to get rid of the "Discussion" issue template. My thought was that "Discussions" (like this one) don't necessarily tie to direct code changes. I think that all of our issues should be directly tied to perceivable changes in the game and code, and ideally, it is a nice, reasonably like one to two hour chunk of work. So my idea was to have a Github team in our repository called "Fire and Ice Team" and then on this team we would create various discussions to deal with elements that just aren't really issues. For example, we have an issue called "Items!" but this is more of an ongoing discussion than it is an issue and it is too big of an idea to be simply completed. On our team page, we might create a discussion called "Items", or maybe something more broad like "New Content Ideas". Then, whenever we are discussing something related to that topic, we would do it there. Similarly, for discussion related to Github itself, we could have another discussion called something like "Repository Changes". Thoughts on this idea?
